Public/Src/IDE/Debugger/Renderer.cs

```diff
@@ -9,21 +9,17 @@
 using System.Globalization;
 using System.Linq;
 using System.Reflection;
-using BuildXL.FrontEnd.Script.Ambients;

```

Comment:
Modulo some trivial refactoring, this file is also pretty much unchanged except that DScript-specific bits were extracted into `DScriptDebuggerRenderer.cs`.